.github/actions/build_contract_artifacts/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
         repository: smartcontractkit/chainlink-solana
         ref: ${{ inputs.ref }}
     - name: Setup go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
       with:
         go-version-file: "go.mod"
```

Comment:
nit: isn't this the default value?

Reply:
Doesn't look like it: https://github.com/actions/setup-go/blob/main/action.yml#L7-L8

Reply:
hmm, but doesn't this suggest it will find it?
https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file